### PR TITLE
Add rb entry

### PR DESCRIPTION
**Pull request by elliezub**
### Description

I created a new semantic HTML entry for the `<rb>` tag. Also I uploaded 2 relevant images to the media folder.
Closes issue [Codecademy#2763](https://github.com/Codecademy/docs/issues/2763)

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/html/concepts/semantic-html/terms/rb/rb.md
Comment (CBID2) [inline content/html/concepts/semantic-html/terms/rb/rb.md:29]: [Proofreading] You forgot to add the syntax section(see my suggestion below). This is something you must always do when creating an entry. See the [term entry template](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md) for reference. 
```suggestion
## Syntax

```html
<ruby>
<rp> (</rp> <rt> <--text goes here --></rt> <rp>) </rp>  
</ruby> 

## Example

The example below showcases how the `<rb>` tag is used to transliterate the Chinese characters meaning 'I like to write code' inside a `<ruby>` element.

```html
<ruby>
  <rb>我</rb><rt>I</rt>
  <rb>喜欢</rb><rt>like</rt>
  <rb>写</rb><rt>to write</rt>
  <rb>程序</rb><rt>code</rt>
</ruby>
```

File: content/html/concepts/semantic-html/terms/rb/rb.md
Comment (CBID2) [inline content/html/concepts/semantic-html/terms/rb/rb.md:45]: [Proofreading] [The style guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#code) suggests using the language's name when writing code snippets. 
```suggestion
```html
<ruby>
  <rb>我</rb><rt>wǒ</rt>
  <rb>喜欢</rb><rt>xǐ huān</rt>
  <rb>写</rb><rt>xiě</rt>
  <rb>程序</rb><rt>chéng xù</rt>
</ruby>
```
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @elliezub! :) I pointed out some errors in my inline comments, so check them out. In the future, consider reading the boilerplate text in the term entry template before submitting it as a PR. Other than that, you're off to a great start! :)

File: content/html/concepts/semantic-html/terms/rb/rb.md
Comment (CBID2) [inline content/html/concepts/semantic-html/terms/rb/rb.md:15]: [Proofreading] Always hyperlink elements that correlate to the one you're writing about. 
```suggestion
The **`<rb>`** element designates the base text for ruby annotations in East Asian languages. It must be within `<ruby>` tags and is often paired with [`<rt>`](https://www.codecademy.com/resources/docs/html/semantic-html/rt) for pronunciation guides and `<rp>` for fallback rendering.
```

File: content/html/concepts/semantic-html/terms/rb/rb.md
Comment (CBID2) [inline content/html/concepts/semantic-html/terms/rb/rb.md:17]: [Proofreading] Consider naming the element that replaced the deprecating one.
```suggestion
> **Note:** The `<rb>` tag is considered deprecated, meaning it's no longer recommended for use in new projects. What is used now is `<ruby>`.
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
I added some more changes @elliezub

**Comment by elliezub** [comment]
Ok thank you Chrissy! I will review the new changes and update accordingly :)

File: content/html/concepts/semantic-html/terms/rb/rb.md
Comment (CBID2) [inline content/html/concepts/semantic-html/terms/rb/rb.md:41]: [nitpick] I recommend making the image smaller so it can fit into the entry. Here's what I use to help me with that: https://imageresizer.com/
```suggestion
![Output of the above code with English translations](https://raw.githubusercontent.com/Codecademy/docs/main/media/iliketowritecode.png)
```

File: content/html/concepts/semantic-html/terms/rb/rb.md
Comment (CBID2) [inline content/html/concepts/semantic-html/terms/rb/rb.md:54]: [nitpick] See my feedback [on this line](https://github.com/Codecademy/docs/pull/2897/files#r1278608779). They share the same issue. 
```suggestion
![Output of the above code with Pinyin transliterations](https://raw.githubusercontent.com/Codecademy/docs/main/media/woxihuanxiechengxu.png)
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Just added some more changes

**Comment by elliezub** [comment]
Thanks Chrissy! I've updated everything based on the changes you suggested.

**Review by CBID2 — APPROVED** [review]
Looks good @elliezub! :) @SSwiniarski, she's ready for a review now! :)

File: content/html/concepts/semantic-html/terms/rb/rb.md
Comment (SSwiniarski) [inline content/html/concepts/semantic-html/terms/rb/rb.md:9]:
```suggestion
  - 'Tags'
  - 'Obsolete'
```
We have an "Obsolete" tag for cases like this where the item is depreciated.

**Review by SSwiniarski — APPROVED** [review]
@elliezub Aside from a tag I'd like you to add, it looks good for a second maintainer review.

File: content/html/concepts/semantic-html/terms/rb/rb.md
Comment (elliezub) [reply content/html/concepts/semantic-html/terms/rb/rb.md:9]: Great! Thank you :) Will remember that for next time.

**Comment by elliezub** [comment]
@KTom101 this PR is ready for a second review :)

**Comment by elliezub** [comment]
Thank you @yangc95 😄

**Pull request merged**
